salt/states/zenoss.py

```diff
@@ -0,0 +1,71 @@
+# -*- coding: utf-8 -*-
+'''
+State to manage monitoring in Zenoss.
+
```

Comment:
Sure, would that be 2015.8 right now? Sorry, just not familiar with Salt contributions.